scripts/gha/print_matrix_configuration.py

```diff
@@ -70,7 +70,7 @@
       }
     },
     "config": {
-      "platform": "Windows,macOS,Linux,Android,iOS",
+      "platform": "Windows,macOS,Linux",
```

Comment:
why we don't need Android and iOS here?

Reply:
Because these 2 platforms are flaky and takes a long time to run.
I'll have a follow up PR to enable parallel building for these 2 platforms.